docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '2.2'
+
+volumes:
+  boot:
+  state:
+  data:
+
+services:
+  provision:
+    image: alpine:3.13
+    command:
+      - sh
+      - -c
+      - |
+        if ! [ -f /mnt/boot/config.json ]; then
+          cp /config.json /mnt/boot/config.json
+        else
+          echo "INFO: Reusing already existing config.json in docker volume."
+        fi
+    volumes:
+      - boot:/mnt/boot
+      - ./container-fleet.config.json:/config.json
+  os:
+    build:
+      context: .
+      dockerfile: Dockerfile.template
+      args:
+        OS_VERSION: 2.68.1_rev1
+        DEVICE_TYPE: intel-nuc
```

Comment:
do we want to have this default?

Reply:
It's most likely to be run from an x86 PC, so I think it's sensible. Again, ideally we would make the hostOS app compose file be this in future, so the device type will be implicit.